Tests/Integration/DSCResources/McAfee.config.ps1

```diff
@@ -0,0 +1,79 @@
+configuration McAfee_config
+{
+    param
+    (
+
+        [Parameter()]
+        [AllowNull()]
+        [string]
+        $TechnologyRole,
+        
+        [Parameter()]
+        [AllowNull()]
+        [string]
+        $TechnologyVersion,
+
+        [Parameter(Mandatory = $true)]
+        [string]
+        $StigVersion,
+
+        [Parameter()]
+        [string[]]
+        $SkipRule,
+
+        [Parameter()]
+        [string[]]
+        $SkipRuleType,
+
+        [Parameter()]
+        [hashtable]
+        $Exception,
+
+        [Parameter()]
+        [hashtable]
+        $BackwardCompatibilityException,
+
+        [Parameter()]
+        [object]
+        $OrgSettings
+    )
+
+    Import-DscResource -ModuleName PowerStig
+
+    Node localhost
```

Comment:
[All PowerShell keywords should be lowercase](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md#correct-format-for-keywords)